src/groupWith.js

```diff
@@ -0,0 +1,33 @@
+var _curry2 = require('./internal/_curry2');
+
+/**
+ * Takes a list and returns a list of lists where each sublist contains only equal elements.
+ *
+ * @func
+ * @memberOf R
+ * @category List
+ * @sig (a, a -> Boolean) -> [a] -> [[a]]
+ * @param {Function} fn Function :: a, a -> Boolean
```

Comment:
OK, I don't 100% understand which of these `@param` descriptions are required or what's expected in them. I based this docstring off of [`groupBy`](https://github.com/ramda/ramda/blob/35f9094d94865d1b368d4a8399223a0261508a3d/src/groupBy.js#L22)